### DISCUSSION TITLE
Bref v2 CLI - variable resolution

### DISCUSSION THREAD

**Discussion by ArronKing** [brefphp/bref#1517] [Support]
I've been trying to upgrade to bref v2, but I'm struggling with the CLI. I used to run something like:

`vendor/bin/bref cli --region=eu-west-2 connectors-prod-artisan -- migrate --force` to run the migrations

So v2 that's now:

`serverless bref:cli --args="migrate --force"`

But I'm getting the following error:

```
Cannot resolve serverless.yml: Variables resolution errored with:
- Cannot resolve variable at "provider.vpc.subnetIds.0": Value not found at "cf" source,
- Cannot resolve variable at "provider.environment.DB_DATABASE": Value not found at "ssm" source,
```

Is there a different way I should be loading variables from SSM / CloudFront. Or is there a different way I can trigger the cli? Invoking Lambda directly does work, so I can use that. But the cli command was a bit nicer to use.

Thanks in advance.

**Comment by mnapoli** [comment]
Is it a stage issue? E.g. missing `--stage=prod` option?

**Comment by ArronKing** [comment]
It wasn't, but your suggestion made me rethink what was going on. I have multiple AWS accounts on my machine, using the --aws-profile option did resolve this.

Thanks for your response, really appreciated. Now resolved.
